DependencyInjection/FOSHttpCacheExtension.php

```diff
@@ -84,19 +84,13 @@ public function load(array $configs, ContainerBuilder $container)
             $loader->load('cache_manager.xml');
         }
 
-        $container->setParameter($this->getAlias().'.compiler_pass.tag_annotations', $config['tags']['enabled']);
-        if ($config['tags']['enabled']) {
-            // true or auto
-            $container->setParameter($this->getAlias().'.tag_handler.header', $config['tags']['header']);
-            $loader->load('tag_listener.xml');
-            if (!empty($config['tags']['rules'])) {
-                $this->loadTagRules($container, $config['tags']['rules']);
-            }
-
-            $tagsHeader = $config['tags']['header'];
-            $container->getDefinition($this->getAlias().'.cache_manager')
-                ->addMethodCall('setTagsHeader', array($tagsHeader))
-            ;
+        if (isset($config['proxy_client'], $config['tags'])) {
+            $this->loadCacheTagging(
+                $container,
+                $loader,
+                $config['tags'],
+                $this->getDefaultProxyClient($config['proxy_client'])
```

Comment:
tagging would have to work with the default client - tag handler is only wired with the default proxy client (via the cache manager).